modules/osu_migrations_files/migrations/upgrade_d7_files.yml

```diff
@@ -35,11 +35,11 @@ source:
     #
     # See the configuration for the source_full_path property in the process
     # section below.
-    source_base_path: '/var/www/html/docroot'
+    source_base_path: 'https://health.oregonstate.edu'
```

Comment:
Is there a way we can get this programmatically?

Reply:
we should leave the source path as /var/www/html/docroot and only change it on a case-by-case migration.
At some point we will want to also migrate private files and those have to be available on the file system to migrate.